.travis.yml

```diff
@@ -2,10 +2,18 @@ language: c++
 sudo: false
 env:
   - NODE_VERSION="0.10"
+  - NODE_VERSION="0.11"
   - NODE_VERSION="0.12"
+  - NODE_VERSION="iojs-v1"
```

Comment:
We intentionally dropped v0.11 support because it's supposed to be compatible with v0.12. See #307.

Artur took out iojs support a month or so ago. Didn't iojs and node merge, so there's no reason to test against these versions? I could be mistaken, so please correct me if I'm wrong.